apps/wallet/src/ui/app/staking/stake/UnstakeForm.tsx

```diff
@@ -57,7 +63,23 @@ export function UnStakeForm({
                             weight="medium"
                             color="steel-darker"
                         >
-                            Your Stake
+                            Current Epoch Ends
+                        </Text>
+                        <div className="flex gap-0.5 items-end">
```

Comment:
is the `gap` needed here since there's only one flex child here? or if you're just trying to move the text to the far right, you could probs just do `ml-auto` and forgo the flex styling